pkg/workflows/upgrade.go

```diff
@@ -393,7 +393,7 @@ func (s *updateClusterAndGitResources) Run(ctx context.Context, commandContext *
 	}
 
 	logger.Info("Resuming EKS-A controller reconciliation")
-	err = commandContext.ClusterManager.ResumeEKSAControllerReconcile(ctx, target, commandContext.ClusterSpec, commandContext.Provider)
+	err = commandContext.ClusterManager.ResumeEKSAControllerReconcile(ctx, target, commandContext.CurrentClusterSpec, commandContext.Provider)
```

Comment:
doesn't this happen after the upgrade? and in that case, shouldn't it be `ClusterSpec`?